crypto/build_dependency.sh

```diff
@@ -14,7 +14,7 @@ fi
 rm -rf "${RELIC_DIR}"
 
 # relic version or tag
-relic_version="9206ae50b667de160fcc385ba3dc2c920143ab0a"
+relic_version="9a0128631841c7ade82460e8e80f8289cf9120b5"
```

Comment:
Nit: I think we could take in 3dd26f5ae28234e02fa4da375422ba954b7b1e53

Reply:
Thanks for listing the changelog!

That's what I tried first, but our validation fails with that Relic version and I didn't want to go down the path of integrating the new version and solving the issues.

Reply:
That's interesting! Do we have an issue for a relic update where you could add a few words on that failing validation?

Reply:
There isn't an issue for a Relic upgrade yet. I usually look at the Relic repo updates and only create an upgrade task when there are changes/improvements I would like our lib to use.

Reply:
How about creating one on the open-source repo?
- that would help capture your experience in witnessing a validation failure,
- we can consider an OSS contribution that would help us with the upgrade.

Reply:
It's a good idea , but either way (issue on our repo or issue on relic's), I would need to investigate the crash a bit. This is not a complicated work, but is is not causing a blocker for us now, and it should come behind other priorities. We're gonna do this work at some point anyway when there will be the need to upgrade Relic. What so you think?